app/core/templates/core/profile/index.html

```diff
@@ -16,3 +16,11 @@
     <hr>
     {% include 'core/profile/questions.html' %}
 {% endblock %}
+
+{% block 'meta' %}
+    <meta property="og:title" content="{{politician.first_name}} {{politician.last_name}}" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="{{politician.image}}" />
```

Comment:
Wouldn't this return the ImageField?
`{{politician.image.url}}` should work but will return the url without the baseurl.